api/app/controllers/api/v1/applications_controller.rb

```diff
@@ -47,6 +54,22 @@ def update
             end
         end
 
+        # /applications/delete or /applications/:id/delete
+        def delete
+            params.require(:id)
+            application = Application.find(params[:id])
+            if application
+                applicant_data_for_matching = ApplicantDataForMatching.find_by(
```

Comment:
Another place to potentially change to `application.applicant_data_for_matching`

Reply:
I'll fix this as well. Thanks.

Reply:
Done.